### PR TITLE
RMC-157 Created new test for scheduled reminder questionnaires

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context
Added test for scheduled reminder questionnaires

# What has changed
- Added new print file reminder questionnaire test
# How to test?
- Run acceptance tests with branches on [Action scheduler ](https://github.com/ONSdigital/census-rm-action-scheduler/pull/32)and [print-file service](https://github.com/ONSdigital/census-rm-print-file-service/pull/13)

# Links
[Trello](https://trello.com/c/4GUC0HS0/)

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[RMC-157 Schedule, generate and distribute reminder PQs (5)](https://trello.com/c/4GUC0HS0/723-rmc-157-schedule-generate-and-distribute-reminder-pqs-5)

File: acceptance_tests/features/steps/case_events.py [hunk 1]
```diff
@@ -76,7 +88,7 @@ def _test_cases_correct(context):
                 del context.expected_sample_units[index]
                 break
         else:
-            assert False, 'Could not find sample unit'
+            tc.fail(msg='Could not find sample unit')
```
Comment (AdamHawtin) [inline acceptance_tests/features/steps/case_events.py:91]: 👍

File: acceptance_tests/utilities/print_file_helper.py [hunk 2]
```diff
@@ -127,6 +154,23 @@ def _create_expected_questionnaire_csv_line(case, prefix):
     )
 
 
+def _create_expected_reminder_questionnaire_csv_line(case, prefix):
```
Comment (AdamHawtin) [inline acceptance_tests/utilities/print_file_helper.py]: I think this is duplicating the existing `_create_expected_questionnaire_csv_line`

File: acceptance_tests/features/print_file.feature [hunk 3]
```diff
@@ -41,3 +41,16 @@ Feature: Scheduled print and manifest files can be generated and uploaded
       | pack code     | questionnaire types | number of matching cases | sample file                          |
       | P_RL_1RL1_1   | [01]                | 2                        | sample_input_england_census_spec.csv |
       | P_RL_2RL2B_3a | [02]                | 2                        | sample_input_wales_census_spec.csv   |
+
+
+  Scenario Outline:  Generate print files and log events for scheduled questionnaire letters
+    Given sample file "<sample file>" is loaded successfully
```
Comment (AdamHawtin) [inline acceptance_tests/features/print_file.feature:47]: 👍 Nice to have a single step for successful sample load/case distribution.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
All passing locally, just a minor code comment.

File: acceptance_tests/utilities/print_file_helper.py (hunk 2 above)
Comment (ryangrundy7) [reply acceptance_tests/utilities/print_file_helper.py]: Removed

**Review by hbrace — APPROVED** [review]

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**